.travis.yml

```diff
@@ -18,7 +19,7 @@ addons:
     repo_token: 806ffaf1bcd1080b8b6ba4839e7acefdb6508332b18162816e4683307e008fec
 
 install:
-  - composer install
+  - composer update
```

Comment:
The `composer install` command will install cached dependencies because the `composer.lock` file is under Git version control.

The `composer update` will update the cached dependencies to be compatible with `php-7.4` version.